tests/unit/clustering/kmeans_unittest.cc

```diff
@@ -11,9 +11,20 @@
 #include <shogun/clustering/KMeans.h>
 #include <shogun/clustering/KMeansMiniBatch.h>
 #include <shogun/distance/EuclideanDistance.h>
+#include <shogun/lib/observers/ParameterObserver.h>
+#include <shogun/lib/observers/ParameterObserverLogger.h>
 
 using namespace shogun;
 
+void check_consistency_observable(const CKMeans * kmeans, ParameterObserver * observer)
```

Comment:
@karlnapf I've updated one KMean unit test to check also if the observed values received are actually okay (such to avoid also to add a new copy-pasted test). Not sure if it is worth to update the other unit tests though...